src/widgets/study-with-us/ui/study-with-us.module.scss

```diff
@@ -0,0 +1,77 @@
+.stud-with-us {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+
+  @include media-laptop {
+    gap: 48px;
+  }
+}
+
+.study-wrap {
+  display: flex;
+  flex-direction: row;
+  gap: 50px;
+  align-items: flex-start;
+  justify-content: space-between;
+
+  @include media-tablet-large {
+    flex-direction: column;
+    gap: 40px;
+  }
+}
+
+.study-general {
+  width: 640px;
+  text-align: left;
+
+  @include media-tablet-large {
+    width: 100%;
+  }
+}
+
+.picture {
+  width: 347px;
+  height: 327px;
+
+  @include media-tablet {
+    width: 278px;
+    height: auto;
+    object-fit: contain;
+  }
+
+  @include media-tablet-large {
+    align-self: center;
+  }
+}
+
+.study-options {
+  display: flex;
+  flex-direction: row;
+  gap: 24px;
+  align-items: flex-start;
+
+  @include media-tablet-large {
+    flex-direction: column;
+    gap: 32px;
+  }
+
+  @include media-tablet {
+    gap: 28px;
+  }
+}
+
+.option-item {
```

Comment:
With the current implementation, there may be extra empty space on the right
![image](https://github.com/user-attachments/assets/a9bc5366-c938-4cd2-97bc-4dec856b36bd)

Reply:
Done